llvm/test/CodeGen/AArch64/neon-partial-reduce-dot-product.ll

```diff
@@ -1,6 +1,7 @@
 ; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 5
-; RUN: llc -mtriple aarch64 -mattr=+neon,+dotprod < %s | FileCheck %s --check-prefixes=CHECK,CHECK-DOT
-; RUN: llc -mtriple aarch64 -mattr=+neon < %s | FileCheck %s --check-prefixes=CHECK,CHECK-NODOT
+; RUN: llc -mtriple aarch64 -mattr=+neon,+dotprod,+i8mm < %s | FileCheck %s --check-prefixes=CHECK,CHECK-DOT,CHECK-I8MM
+; RUN: llc -mtriple aarch64 -mattr=+neon,+i8mm < %s | FileCheck %s --check-prefixes=CHECK,CHECK-NODOT,CHECK-I8MM
+; RUN: llc -mtriple aarch64 -mattr=+neon,+dotprod < %s | FileCheck %s --check-prefixes=CHECK,CHECK-DOT,CHECK-NOI8MM
```

Comment:
This hasn't worked as expected because there are no `CHECK-I8MM` lines. I guess I was expecting too much from the auto-update script?

Reply:
I think I see what has happened.  I had assumed `+i8mm` would enable `usdot` instructions but the DAG combine requires both `+dotprod` and `+i8mm` for that to happen.

Reply:
That's nice. I re-ordered the run lines a little to reduce the number of total changes.